docs/sources/ocimetrics.md

```diff
@@ -2,21 +2,29 @@
 
 This event source collects metrics data from the [Oracle Cloud](https://cloud.oracle.com).
 
-## Prerequisites
+## Prerequisite(s)
 
-An Oracle Cloud account with permissions to inspect and read metrics for the
-Oracle Cloud Infrastructure (OCI) compartment is required.  In addition, the
-account must have the account API secret added.
+- Oracle Cloud Account
+- Oracle Cloud Infrastructure (OCI)
+- Oracle Cloud Secret
+
+### Oracle Cloud Account
+
+An Oracle Cloud account is required.
+
+### Oracle Cloud Infrastructure (OCI)
+
+The Oracle Cloud account needs to have permissions to inspect and read metrics for the Oracle Cloud Infrastructure (OCI) compartment is required. In addition, the account must have the account API secret added.
```

Comment:
Maybe we can improve phrasing here. The secret is taking care of in the next paragraph, it can be removed.

```suggestion
The Oracle Cloud account needs to have permissions to inspect and read metrics for the Oracle Cloud Infrastructure (OCI) compartment.
```